lib/internal/errors.js

```diff
@@ -1104,18 +1104,28 @@ E('ERR_INVALID_PACKAGE_CONFIG', (path, message, hasMessage = true) => {
 }, Error);
 E('ERR_INVALID_PACKAGE_TARGET',
   (pkgPath, key, subpath, target, base = undefined) => {
+    const relError = typeof target === 'string'
+      && target.length && !target.startsWith('./');
     if (key === null) {
       if (subpath !== '') {
         return `Invalid "exports" target ${JSONStringify(target)} defined ` +
         `for '${subpath}' in the package config ${pkgPath} imported from ` +
-        base;
+        `${base}.${relError ? ' - targets must start with "./"' : ''}`;
       } else {
         return `Invalid "exports" main target ${target} defined in the ` +
-          `package config ${pkgPath} imported from ${base}.`;
+          `package config ${pkgPath} imported from ${base}.${relError ?
+          ' - targets must start with "./"' : ''}`;
       }
     } else if (key === '.') {
       return `Invalid "exports" main target ${JSONStringify(target)} defined ` +
-      `in the package config ${pkgPath}${sep}package.json`;
+      `in the package config ${pkgPath}${sep}package.json${relError ?
```

Comment:
Should paths be quoted?

Reply:
This was not a change I made in this PR. I think if we want to improve or make the paths consistent that should come as a follow up

Reply:
As a general rule I personally aim to quote specifiers and not absolute paths since they are underlined by the editor anyway.

Reply:
@guybedford, are you referring to VS Code's integrated console?

Reply:
Yes exactly.